internal/config/config.go

```diff
@@ -2,18 +2,22 @@
 package config
 
 import (
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
 	"go.infratographer.com/x/echojwtx"
 	"go.infratographer.com/x/echox"
 	"go.infratographer.com/x/events"
 	"go.infratographer.com/x/loggingx"
 	"go.infratographer.com/x/otelx"
+	"go.infratographer.com/x/viperx"
 
 	"go.infratographer.com/permissions-api/internal/spicedbx"
 )
 
 // EventsConfig stores the configuration for a load-balancer-api events config
 type EventsConfig struct {
-	Subscriber events.SubscriberConfig
+	events.Config `mapstructure:",squash"`
+	Topics        []string
```

Comment:
Looks like we do actually have a config section for event topics; we should use this, right?

Reply:
indeed!